projects/batfish/src/main/java/org/batfish/representation/juniper/JuniperConfiguration.java

```diff
@@ -92,6 +96,8 @@ private static String communityRegexToJavaRegex(String regex) {
 
   private final Map<String, BaseApplication> _applications;
 
+  private final NavigableMap<String, AuthenticationKeyChain> _authenticationKeyChains;
```

Comment:
This is supposed to be a Juniper-specific structure. Conversion to vendor-independent should occur in `toVendorConfiguration()`